src/plugins/intel_cpu/thirdparty/CMakeLists.txt

```diff
@@ -60,15 +60,14 @@ function(ie_add_onednn)
         endif()
 
         set(ARM_COMPUTE_SCONS_JOBS "8" CACHE STRING "Number of parallel threads to build ARM Compute Library")
+        set(DNNL_USE_ACL ON CACHE BOOL "" FORCE)
```

Comment:
As we agreed we also need to update "OV_CPU_WITH_ACL" initalization condition to enable ACL based ops on plugin level

Reply:
Good catch! Fixed